src/cool_seq_tool/mappers/exon_genomic_coords.py

```diff
@@ -217,6 +229,7 @@ async def genomic_to_transcript_exon_coordinates(
         end: Optional[int] = None,
         strand: Optional[Strand] = None,
         transcript: Optional[str] = None,
+        is_fusion_transcript_segment: Optional[bool] = None,
```

Comment:
Do this in all occurrences 

```suggestion
        is_fusion_transcript_segment: bool = False,
```